ecommerce/models.py

```diff
@@ -247,6 +248,20 @@ def course_keys(self):
             raise ImproperlyConfigured("content_object expected to be one of Program, Course, CourseRun")
 
     @property
+    def program(self):
+        """Get the program which all course keys for the coupon belong to"""
```

Comment:
Nit: not sure I understand what this docstring means. 
I know what this method does, but I do not think that this is the right way to explain it.